README.md

```diff
@@ -31,4 +31,19 @@ Then import it into python.
 ## Testing
 
 This package uses [pytest](https://pytest.org/) for tests.
-To run tests locally, execute the command `pytest` from the command line.
+To run tests locally, execute the command `pytest` from the command line:
+
+```
+>>>pytest
```

Comment:
pls lemme know if this didn't work as expected - ie added a small commit here?

Reply:
Fixed this too